src/main/java/com/cozyhome/onlineshop/reviewservice/service/ReviewServiceImpl.java

```diff
@@ -29,7 +30,8 @@ public class ReviewServiceImpl implements ReviewService{
     public List<ReviewAdminResponse> getReviews() {
         List<Review> reviews = repository.findAll();
         if (reviews.isEmpty()) {
-            throw new DataNotFoundException("The are no reviews.");
+            log.info("[ON getReviews]:: The are no reviews for any product.");
+            return new ArrayList<>();
         }
```

Comment:
instead of this if statement we can add filter method to stream to check null values:
"return reviews.stream().filter(review -> review != null).map(review -> mapper.map(review, ReviewAdminResponse.class)).toList();" if the reviews List is empty will be returned empty list

Reply:
done